### PR TITLE
Add Eventpage

### LINKED ISSUE #35: Finish Eventhome-Page
# User story
As a **user**,
I want to see different lists of tasting events
so I can get an overview.

# Description

# Notes / Links

# Acceptance Criteria
- [x] For your Taste List
- [x] By beverage / by country list
- [x] Click footer icon --> change color
- [x] Improve border icons

### PR DESCRIPTION
**Pull request by rheimers**
Fix #35 

Bildschirmfoto 2020-09-29 um 19 45 15

- Searchbar / Header imported
- Footer imported 
- EventList and FilterLists imported
- Fetch data

### CODE REVIEW

File: client/src/GlobalStyles.js [hunk 1]
```diff
@@ -16,7 +16,11 @@ const GlobalStyles = () => {
         }
         :root {
           --clr-primary: #ffa200;
-          --highlight-clr-category: #fa20d8f2;
+          --highlight-clr-category1: #fa20d8;
+          --highlight-clr-category2: #0bff69;
+          --highlight-clr-category3: #fbff14;
+          --highlight-clr-category4: #ffa200;
+          --highlight-clr-category5: #33b7f2;
```
Comment (LebkuchenCJ) [inline client/src/GlobalStyles.js:23]: Is this really part of this PR or could this be handled in a global PR?

File: client/src/api/getEvents.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export const getEvents = async () => {
+  const response = await fetch("/api/events");
+  if (!response.ok) {
+    throw response;
+  }
+
+  const result = await response.json();
+  console.log(result);
```
Comment (LebkuchenCJ) [inline client/src/api/getEvents.js:8]: Do you still need this console.log? If not, you could remove it

File: client/src/components/EventCard.js [hunk 3]
```diff
@@ -14,11 +14,28 @@ const Card = styled.div`
   height: 93px;
   border-radius: 12px;
```
Comment (LebkuchenCJ) [inline client/src/components/EventCard.js:15]: Also this is not part of your change, you should keep in mind to not work with fixed PX's

File: client/src/components/EventCard.js [hunk 4]
```diff
@@ -14,11 +14,28 @@ const Card = styled.div`
...
+        default:
+          return "var(--font-color-white)";
+      }
+    }};
+
     grid-row: 3;
     grid-column: 1 / 5;
-    font-family: var(--font-archivoblack);
-    font-size: 12px;
+    font: 11px/9px var(--font-archivoblack);
```
Comment (LebkuchenCJ) [inline client/src/components/EventCard.js:37]: Below you are working with EM. Why not here?

File: client/src/components/FilterList.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  }
+`;
+
+const ListContainerScroller = styled.div`
+  width: 100%;
+  overflow: auto;
+`;
+const ListContainer = styled.div`
+  display: flex;
+  & > *:not(:first-of-type) {
```
Comment (LebkuchenCJ) [inline client/src/components/FilterList.js:25]: Nice. Never knew this is possible 👍

File: client/src/pages/EventPage.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  flex-flow: column;
+  margin-left: 0.8em;
+`;
+
+function EventPage() {
+  const { data: events } = useAsync(getEvents);
+  console.log(events);
+  return (
+    <Container>
+      <Searchbar></Searchbar>
```
Comment (LebkuchenCJ) [inline client/src/pages/EventPage.js:21]:
```suggestion
      <Searchbar/>
```

File: client/src/pages/EventPage.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+const Container = styled.div`
+  display: flex;
+  flex-flow: column;
+  margin-left: 0.8em;
+`;
+
+function EventPage() {
+  const { data: events } = useAsync(getEvents);
+  console.log(events);
```
Comment (LebkuchenCJ) [inline client/src/pages/EventPage.js:18]: Do you still need this console.log?

File: client/src/pages/EventPage.js [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+import styled from "@emotion/styled";
+
+const Container = styled.div`
+  display: flex;
+  flex-flow: column;
+  margin-left: 0.8em;
+`;
+
+function EventPage() {
+  const { data: events } = useAsync(getEvents);
```
Comment (LebkuchenCJ) [inline client/src/pages/EventPage.js:17]: Why don't you import the ERROR and LOADING params from your hook here?

**Review by LebkuchenCJ — COMMENTED** [review]
Hey, GJ 👍 

Some comments:
Only found some little things. See in your comments.
Also, you could assign yourself and the project to keep track in you kanban board: 
image

File: client/src/GlobalStyles.js (hunk 1 above)
Comment (rheimers) [reply client/src/GlobalStyles.js:23]: you are totally right - should have been in the eventcard branch

File: client/src/api/getEvents.js (hunk 2 above)
Comment (rheimers) [reply client/src/api/getEvents.js:8]: deleted

File: client/src/components/EventCard.js (hunk 3 above)
Comment (rheimers) [reply client/src/components/EventCard.js:15]: dito

File: client/src/components/EventCard.js (hunk 4 above)
Comment (rheimers) [reply client/src/components/EventCard.js:37]: initially started with px, going to convert everything into rem

File: client/src/components/FilterList.js (hunk 5 above)
Comment (rheimers) [reply client/src/components/FilterList.js:25]: thx

File: client/src/pages/EventPage.js (hunk 8 above)
Comment (rheimers) [reply client/src/pages/EventPage.js:17]: well, you gonna need to start somewhere ;-). will fix this

File: client/src/pages/EventPage.js (hunk 7 above)
Comment (rheimers) [reply client/src/pages/EventPage.js:18]: deleted

File: client/src/pages/EventPage.js (hunk 6 above)
Comment (rheimers) [reply client/src/pages/EventPage.js:21]: thx

**Pull request merged**